package.json

```diff
@@ -39,16 +39,19 @@
   },
   "homepage": "https://github.com/dashevo/wallet-lib#readme",
   "dependencies": {
-    "@dashevo/dapi-client": "~0.15.0-dev.1",
+    "@dashevo/dapi-client": "~0.15.0-dev.4",
     "@dashevo/dashcore-lib": "~0.18.11",
+    "@dashevo/grpc-common": "~0.3.0",
     "cbor": "^5.0.2",
     "crypto-js": "^4.0.0",
     "lodash": "^4.17.19",
     "pbkdf2": "^3.1.1",
     "winston": "^3.2.1"
   },
   "devDependencies": {
+    "@dashevo/dapi-grpc": "^0.14.0",
```

Comment:
It seems we are not using it anywhere.

Reply:
We were going to use it in the integration test to test the error, but there seems to be no easy way to reuse the errors, so we went with mocks. This indeed needs to be removed

Reply:
Done